Library/Homebrew/cmd/bottle.rb

```diff
@@ -300,69 +301,78 @@ def bottle_formula(f)
     puts "./#{filename}"
     puts output
 
-    if ARGV.include? "--rb"
-      File.open("#{filename.prefix}.bottle.rb", "w") do |file|
-        file.write("\# #{f.full_name}\n")
-        file.write(output)
+    if ARGV.include? "--json"
+      json = {
+        f.full_name => {
+          "formula" => {
+            "pkg_version" => f.pkg_version.to_s,
+            "path" => f.path.to_s.gsub("#{HOMEBREW_REPOSITORY}/", ""),
```

Comment:
This likely needs to be relative to `f.tap.path` instead of `HOMEBREW_REPOSITORY`.